application/src/test/resources/server/cert.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDLDCCAhQCCQDwyqHW1MWFjjANBgkqhkiG9w0BAQsFADBWMQswCQYDVQQGEwJU
+RTENMAsGA1UECAwEVEVTVDENMAsGA1UEBwwEVEVTVDENMAsGA1UECgwEVEVTVDEN
+MAsGA1UECwwEVEVTVDELMAkGA1UEAwwCQ0EwIBcNMjEwMTIxMTIwNTIwWhgPMjE4
+NTA1MDExMjA1MjBaMFgxCzAJBgNVBAYTAlRFMQ0wCwYDVQQIDARURVNUMQ0wCwYD
+VQQHDARURVNUMQ0wCwYDVQQKDARURVNUMQ0wCwYDVQQLDARURVNUMQ0wCwYDVQQD
+DARVc2VyMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA57QgmBI5o6VP
+pNoio0oCbLbSnXfbfZS/wVZ5ET1QOhr+ZpbZqwpZTt5QbSguFQNMJn0wkISa1Ymf
+QBL+SaZHsxGy4SwX0Fs4N0ou1/38iBubSNfl9FZ/3nwYMZ9do5651s8MHbqGcFdY
+oHOtdvVuMKDCpUg2bJZHM4UPya35SZBsjCqatA0xrnDBdjXnPVDoj3O/Y+H6D9pT
+pcBG2MabMyC+GgAvt5sZIJxbYZNchXHFiML6sPUXoxpC5SbKmj1/Da5veNBbQsXw
+C29ofWeM1t0hEtSkaQCSVl1TRCesA9u5AUBfnKnhlzABizRz6Wb/c3jH59GtS2M3
+mBMmOJLfZwIDAQABMA0GCSqGSIb3DQEBCwUAA4IBAQCJ/8Lkh8cR4X0Upmzx+Szv
+YdKvmRM/QC6mtsuvI2qPlwHp3BIIafhiE1ziIIV1K+hqKhq2jN8NTSz8rHVDGMQg
+Bhna4Cc/sIWNfinRZGlo9ITAiJSsgEXmiSYYQwsNOvDbApZVLOgw44SQ7KhEO9bh
+6JUC2ilXA3BLmdyqLQG/ijOdKWBGefO8eL7kZuWQ7kNWywqco72ZjR1woD86JSf4
+2EAO0p4BJ4b8OQ6NlCZZZcQu7tPpwoV4KQ9fgIHS43nwK26T+kKr9PzAVJ8c1S4g
+My4HAU6CWj4m479l+ej6y1pt+VN5ahIKLEpyjBB7JSL23940ZPc1D7pKG4MbdpHj
+-----END CERTIFICATE-----
```

Comment:
No new line at end of file like the other crt files

Reply:
One thing I notice about certificate key pair is that ECDSA type is not accepted, only RSA (with Springboot solution).

https://github.com/spring-projects/spring-boot/blob/v2.7.2/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/server/PrivateKeyParser.java